National data - Completed getHospitals

michaela-marincic opened the pull request:
Not sure if this will affect the open "getHistoricalData" branch that is currently branched off of National Data.

michaela-marincic committed:
Move files to new database folder structure

Move getHospitals.py and getAddressCoords.py to new "National" and "database" folders, respectively.

michaela-marincic committed:
Update getHospitals.py

Clean up data pull and prepare for importing getAddressCoords.

michaela-marincic committed:
Complete getHospitals, make copy of getAddressCoords

This push should finalize the getHospitals function and also makes a copy of getAddressCoords in the National folder called "getAddressCoords_National" to make it easier to call the module/function.

jameson-c commented:
Looks great! I will properly merge to main once I have incorporated this into my broader structure. Only thing is there is a way of doing this without doubling up on getAddressCoords. I will make that change shortly.

jameson-c committed:
DB work from May

Take a look

mihirbhaskar committed:
changing categories

changing the master categories, google search keyword list, adding some basic cleaning of miscategorised assets in Uniontown

mihirbhaskar committed:
adding values master population

mihirbhaskar committed:
updating category descriptions and changing library cat

mihirbhaskar committed:
adding monongahela, comments to resolve duplicate stuff

mihirbhaskar committed:
comment with useful code for location duplicates

mihirbhaskar committed:
community can have their own keywords file

mihirbhaskar committed:
Merge pull request #39 from jameson-c/community-keywords-generalize

community can have their own keywords file

mihirbhaskar committed:
cleaning out more test observations

mihirbhaskar committed:
Merge branch 'main' into national_data
